pkg/operator/resources/cluster/cdiconfig.go

```diff
@@ -73,6 +73,20 @@ func createCDIConfigCRD() *extv1beta1.CustomResourceDefinition {
 										},
 									},
 								},
+								"filesystemOverhead": {
+									Description: "Override the space reserved for filesystem overhead. A value is between 0 and 1, if not defined it is 0.055 (5.5% overhead)",
+									Type:        "object",
+									Properties: map[string]extv1beta1.JSONSchemaProps{
+										"global": {
+											Description: "How much space of a Filesystem volume should be reserved for safety. This value is the global one to be used unless a per-storageClass value is chosen.",
+											Type:        "string",
+										},
+										"storageClass": {
```

Comment:
We need to specify more here about what goes in, in particular we want the pattern on value as well. It should look something like this:
```go
"storageClass": {
  Items: &extv1beta1.JSONSchemaPropsOrArray{
    Schema: &extv1beta1.JSONSchemaProps{
      Description: "Some description explaining what the storage class map does, same as in types.go",
      Type: "array",
      Properties: map[string]extv1beta1.JSONSchemaProps{
      "storageClass": {
        Description: "Name of the storage class",
	Type:        "object",
       },
       "overhead": {
          Description: "The overhead as a decimal",
	  Type:        "object",
          Pattern: "regex for pattern",
	},
    },
```
Not entirely sure this will work since you used a map. But I think the validation is useful